Sources/ConcordiumSwiftSDK/Model.swift

```diff
@@ -52,3 +53,31 @@ struct CryptographicParameters {
     let bulletproofGenerators: String
     let genesisString: String
 }
+
+struct SequenceNumber {
+    var value: UInt64
+
+    init(value: UInt64) {
+        self.value = value
+    }
+
+    /// Get the next sequence number.
+    func next() -> SequenceNumber {
+        return SequenceNumber(value: self.value + 1)
+    }
+
+    /// Increase the sequence number to the next value.
+    mutating func nextMut() {
+        self.value += 1
+    }
```

Comment:
Why have both of these? Or any of them, really? I don't see a use case for them and we can always add them.

Reply:
In general, the way that I approach writing the Swift SDK is just to clone the existing SDKs as I assume that they are working as intended. 

In this case, this type is a clone of rust-version `Nonce` (which should be renamed to SequenceNumber according to Ales). 

Whether these methods are going to be used or not, I have no idea - but shouldn't we strive for as close to a 1:1 copy of existing SDKs as possible? Otherwise, we risk confusing the end users/developers.

Reply:
So the `next_mut` thing makes sense in Rust at least cine it avoids the ugly
```
nonce = nonce.next()
```

It is a small conveniencei mprovement for services which are sending a lot of transactions, of which we have quite a few.

Reply:
Ok, I'll go all the way and nuke the type + associated tests
That also leaves us with a bit less boilerplate code :)